### ISSUE TITLE
Support providing accessibility standard tags and rule options

### ISSUE THREAD

**Issue by jasonkarns** [dequelabs/axe-core#174]
Given a page where
```js
axe.a11yCheck({}, {
  runOnly: { type:"tag", "values": ["wcag2a"] }
}, callback);
```
fails on `image-alt`, `label`, and `link-name`

I would expect that:
```js
axe.a11yCheck({}, {
  runOnly: { type:"tag", "values": ["wcag2a"] },
  rules: { "color-contrast": {"enabled":true} }
}, callback);
```
would *add* the `color-contrast` rule to the checked rules. (`wcag2a` + `color-contrast`). But `color-contrast` isn't checked.

Similarly, Given a page where
```js
axe.a11yCheck({}, {
  runOnly: { type:"tag", "values": ["wcag2a", "wcag2aa"] }
}, callback);
```
fails on `image-alt`, `label`, `link-name`, and `color-contrast`

I would expect that:
```js
axe.a11yCheck({}, {
  runOnly: { type:"tag", "values": ["wcag2a", "wcag2aa"] },
  rules: { "color-contrast": {"enabled":false} }
}, callback);
```
would *skip* the `color-contrast` rule and only fail on `image-alt`, `label`, and `link-name`.

The way I see it, each tag represents a default configuration for a set of rules, enabling some, disabling others. Selecting multiple tags results in the *union* of their enabled rules being run. Individual rule configurations, being more specific than a tag (collection of rules) should be applied in conjunction with the tag(s). (ie, start with the provided tag configuration(s) and then enable/disable any rules specified explicitly via `rules`)

(using http://abcdcomputech.dequecloud.com/ as the test page)

**Comment by WilcoFiers** [comment]
Hi Jason. We are currently testing with just such a feature. Thanks for the comment!
